gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/language/grammar/GenericLiteralVisitor.java

```diff
@@ -30,10 +30,7 @@
 
 import java.math.BigDecimal;
 import java.math.BigInteger;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
```

Comment:
We dont have many code style requirements but one is to not wildcard import. you're IDE probably was the culprit there.

Reply:
Yeah, this was definitely an IDE thing